src/main/kotlin/com/terraformation/backend/tracking/db/ObservationResultsStore.kt

```diff
@@ -352,52 +405,55 @@ class ObservationResultsStore(private val dslContext: DSLContext) {
         .fetch { record ->
           val zones = record[plantingZoneMultiset]
           val species = record[plantingSiteSpeciesMultiset]
-          val liveSpecies =
-              species.filter {
-                it.certainty != RecordedSpeciesCertainty.Unknown &&
-                    (it.totalLive > 0 || it.totalExisting > 0)
-              }
+          val knownSpecies = species.filter { it.certainty != RecordedSpeciesCertainty.Unknown }
+          val liveSpecies = knownSpecies.filter { it.totalLive > 0 || it.totalExisting > 0 }
 
-          var plantingDensity: BigDecimal? = null
-          var totalPlants: BigDecimal? = null
+          var plantingDensity: Int? = null
+          var estimatedPlants: Int? = null
 
           if (zones.isNotEmpty() && zones.all { it.plantingDensity != null }) {
-            val totalArea = zones.sumOf { it.areaHa }
+            plantingDensity =
+                zones
+                    .flatMap { zone ->
+                      zone.plantingSubzones.flatMap { subzone ->
+                        subzone.monitoringPlots.map { plot -> plot.plantingDensity }
+                      }
+                    }
+                    .average()
```

Comment:
Yes, that's the idea. If we ever support variable-size plots, which isn't too likely but could theoretically happen, we'd use each plot's area when calculating the planting density rather than using a constant.